tools/depends/native/Makefile

```diff
@@ -38,36 +55,37 @@ all: native
 	@echo "Dependencies built successfully."
 
 # Dependency layout for parallel builds
+autoconf-archive: autoconf
 autoconf: m4
 automake: autoconf
-autoconf-archive: autoconf
 dpkg: automake gettext libtool pkg-config tar
 flatbuffers: cmake
 heimdal: libtool
-libtool: automake
+JsonSchemaBuilder: automake
 libjpeg-turbo: cmake nasm
 libpng: zlib automake
+libtool: automake
+libtool: automake
```

Comment:
Duplicate

Reply:
haha I told you I hit a rebase with a conflict. clear evidence that this change is needed